app/javascript/plate-picks/components/PlatesSection.vue

```diff
@@ -0,0 +1,78 @@
+<!--
+  Provides a help button which can be clicked to expand into a guide.
+-->
+<template>
+  <section>
+    <h2>Plates</h2>
+    <table class="table table-striped table-sm">
+      <thead>
+        <th scope="col">
+          Barcode
+        </th>
+        <th scope="col">
+          Status
+        </th>
+        <th scope="col">
+          Picks
+        </th>
+      </thead>
+      <tbody>
+        <PlatesSectionPlate
+          v-for="plate in scannedPlates"
+          :key="plate.barcode"
+          v-bind="plate"
+        />
+      </tbody>
+      <tfoot>
+        <td>
+          <input
+            id="scan-plate"
+            ref="scanPlate"
+            v-model="scannedBarcode"
+            class="form-control"
+            @keydown.enter="plateBarcodeScan"
+            @keydown.tab="plateBarcodeScan"
+          >
+        </td>
+        <td colspan="2">
+          <label for="scan-plate">Scan a plate</label>
+        </td>
+      </tfoot>
+    </table>
+  </section>
+</template>
+
+<script>
+import PlatesSectionPlate from './PlatesSectionPlate'
+
+export default {
+  components: {
+    PlatesSectionPlate
+  },
+  data: function () {
+    return {
+      scannedBarcode: ''
+    }
+  },
+  computed: {
+    scannedPlates () { return this.$store.getters.scannedPlates }
+  },
+  mounted() {
+    this.$refs.scanPlate.focus()
+  },
+  methods: {
+    plateBarcodeScan (event) {
```

Comment:
I think at the moment their behaviour is a little too diverged to make it possible to share them, especially with Vuex. Although I will have a check, as it would be good to think about reusable stuff.